src/pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount.tsx

```diff
@@ -108,22 +104,20 @@ function EnableBankAccount({reimbursementAccount, user, onBackButtonPress}) {
                             icon={Expensicons.Close}
                             onPress={BankAccounts.requestResetFreePlanBankAccount}
                             wrapperStyle={[styles.cardMenuItem, styles.mv3]}
-                            disabled={Boolean(pendingAction) || !_.isEmpty(errors)}
+                            disabled={Boolean(pendingAction) || isEmptyObject(errors)}
```

Comment:
Before it checked if it's not empty
```suggestion
                            disabled={Boolean(pendingAction) || !isEmptyObject(errors)}
```